### DISCUSSION TITLE
Appreciation :)

### DISCUSSION THREAD

**Discussion by miversen33** [nvim-java/nvim-java#80] [General]
This is the first Java plugin I have come across for neovim that "just works ™️". Manually setting up jdtls never actually functioned as expected, using nvim-jdtls "worked" but was jank at best.

Yet somehow, adding this and the registry stuff and bam, my LSP is working, my auto complete is functional, inlay hints work, it properly parsed our project structure, everything I would expect my editor to do.

I am not a huge java developer (as I stated in the reddit post for this), so when I need to write java code, I need my editor to "just work". The fact that previously I had to jump into vscode to work on our java code bothered me.

So thank you for this :heart:

**Comment by s1n7ax** [comment]
I'm glad that lot of people find this plugin helpful. Nice to hear that it worked right away. :)
